src/modules/movies.js

```diff
@@ -28,33 +28,38 @@ export default function reducer(state: Object = initialState, action: Object) {
   }
 }
 
-export function receiveMovies(json: Array<number | string> = []) {
+export function receiveMovies(json: Object) {

```

Comment:
Pls, replace it with arrow functions. ES6, etc.

Reply:
Done